### ISSUE TITLE
Provide quick-fix for undeclared symbols

### ISSUE THREAD

**Issue by misode** [SpyglassMC/Spyglass#1222]
When encountering an undeclared symbol like a function or loot table, the LSP should suggest a quick fix for creating that resource.

**Comment by lerokko** [comment]
Wasn't that already suggested 4 years ago, worked in 3.0.0-3.4.19, and just broke in 4.0.0?

https://github.com/SpyglassMC/Spyglass/issues/366

I downgraded to 3.4.19 for the time being and it works. I use it all the time to create mcfunctions. I rarely make them with "New File..." nowadays. This Quick-fix is super useful!

**Comment by MulverineX** [comment]
Yeah this appears to be a regression, apologies! Hopefully someone can work on this soon

**Comment by lerokko** [comment]
> Yeah this appears to be a regression, apologies! Hopefully someone can work on this soon

Should this issue then not receive the bug label?

**Comment by lerokko** [comment]
I do not mean to appear needy or annoying but I am checking this issue weekly since I started to depend on spyglass for my datapacks and heavily used this former feature when it was working. I have no idea how typescript or vscode plugins work otherwise I would offer more help :( but is there any reason why this is labeled as a feature and not a bug?
